datasets/uy/pep/crawler.py

```diff
@@ -26,19 +27,34 @@ def parse_sheet_row(context: Context, row: Dict[str, str]):
     id_number = row.pop("c-i")
     person_name = row.pop("nombre")
     role = row.pop("cargo")
+    organization_name = row.pop("organismo")
 
     person.id = context.make_slug(id_number, prefix="uy-cedula")
     person.add("idNumber", id_number)
     person.add("name", person_name)
 
-    position = h.make_position(context, role, country="uy", lang="spa")
+    position = h.make_position(
+        context, f"{role}, {organization_name}", country="uy", lang="spa"
+    )
+
+    categorisation = categorise(context, position)
+    if not categorisation.is_pep:
+        return
+
+    occupancy = h.make_occupancy(
+        context,
+        person,
+        position,
+    )
```

Comment:
Because this list includes people who are PEPs but not necessarily currently in this role any more, it looks like we need to set no_end_date_implies_current to false in the call to make_occupancy

> As defined by Article 20 of Law No. 19574, PEPs are understood to include individuals who, within the last five years have held or currently hold important public functions, both nationally and internationally.

Reply:
while you're here, could you pass the categorisation value to make_occupancy as well? I haven't merged that documentation yet but it can be handy later.